source/code-generation/query-converter/test-converted-queries/compare-converted-queries.txt

```diff
@@ -1,13 +1,152 @@
-.. _rm-compare-query-results:
+.. _rm-compare-converted-queries
 
-=====================
-Compare Query Results
-=====================
+=========================
+Compare Converted Queries
+=========================
 
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 1
    :class: singlecol
 
-Task page.
\ No newline at end of file
+You can use the :guilabel:`Test Queries` pane to compare source 
+and destination queries, views, and stored procedure results 
+inside of Relational Migrator. The :guilabel:`Test Queries` can 
+help you verify the accuracy of converted code and
+show the source and destination data after you run a sync job.
+
+Before you Begin
+----------------
+
+- You must sign in to your Atlas account in Relational Migrator. 
+  For details, see :ref:`rm-atlas-log-in`.
+
+- For details on how to turn on Query Converter, see 
+  :ref:`rm-enable-query-converter`.
+
+- To use the :guilabel:`Test Queries` pane, Relational
+  Migrator must have access to the Query Runner. For details
+  on how to setup Query Runner, see :ref:`rm-install-query-runner`.
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: Select a query on the :guilabel:`Query Converter` pane
+
+      a. From the :guilabel:`Code Generation` tab, click the 
+         :guilabel:`Query Converter` pane.
+
+         .. note::
+
+           If you are not already logged into your Atlas account or have
+           an expired session, you must login to proceed.
+
+      #. On the left-hand :guilabel:`Queries` pane, select a query, view,
+         or stored procedure.
+
+   .. step:: Open the :guilabel:`Test Queries` pane
+
+      a. From the :guilabel:`Code Generation` tab, click the 
+         :guilabel:`Query Converter` pane.
+
+      #. Click the :icon-lg:`ChevronUp` on the :guilabel:`Test Queries`
+         pane at the bottom-right of the screen.
+
+   .. step:: (Optional) Specify parameters
```

Comment:
It would be nice to have a clearer description of the 3 sections of the test bench: Console, Results and Messages. I'm not sure where this should sit relative to this procedure.

This step is about using the Console. The console is used for running all converted queries, and is also used for running relational stored procedures (but not queries or views). Specifying parameters is one of the things you need to do in the console window.